File: uri/1009/1009.cpp
#include <cstdio>
#include <cmath>

int main(){
    char a[300];
    float b, c;
    scanf("%s", a);
    scanf("%f", &b);
    scanf("%f", &c);
    printf("TOTAL = R$ %.2f\n", b + round((c * 0.15) * 100) / 100);
    return 0;
}

File: solutions/beecrowd/1009/1009.go
package main

import "fmt"

func main() {
	var a string
	var b, c float64

	fmt.Scanf("%s", &a)
	fmt.Scanf("%f", &b)
	fmt.Scanf("%f", &c)

	fmt.Printf("TOTAL = R$ %.2f\n", b+(c*0.15)*100/100)
}
